extended/src/test/java/apoc/load/LoadCsvTest.java

```diff
@@ -448,6 +449,45 @@ public void testLoadCsvNoFailOnError() throws Exception {
                 });
     }
 
+    @Test
+    public void testIssue3156FailOnErrorFalse() {
+        String url = getUrlFileName("faulty.csv").getPath();
+        testResult(db, "CALL apoc.load.csv($url, {failOnError: false})", map("url",url),
+                (r) -> {
+                    Map<String, Object> row = r.next();
+                    assertEquals(0L, row.get("lineNo"));
+                    assertEquals(List.of("Galata Tower", "", "Turkey", "67"), row.get("list"));
+                    row = r.next();
+                    assertEquals(1L, row.get("lineNo"));
+                    assertEquals(List.of("Belem Tower","Lisbon","","30"), row.get("list"));
+                    row = r.next();
+                    assertEquals(2L, row.get("lineNo"));
```

Comment:
```suggestion
                    assertEquals(3L, row.get("lineNo"));
```
Dovrebbe essere 3 non due e cosi via i successivi

Reply:
Cambiato